client/my-sites/site-settings/form-general.jsx

```diff
@@ -359,6 +360,31 @@ export class SiteSettingsFormGeneral extends Component {
 		);
 	}
 
+	hideActionbar() {
+		const { translate, fields, updateFields } = this.props;
+		return (
+			<FormFieldset>
+				<FormLabel htmlFor="site-settings__wpcom_hide_action_bar">
+					{ translate( 'Action Bar visibility' ) }
+				</FormLabel>
+				<ToggleControl
+					id="site-settings__wpcom_hide_action_bar"
+					__nextHasNoMarginBottom
+					label={ translate( 'Hide the Action Bar on the front end of the site.' ) }
+					checked={ !! fields?.wpcom_hide_action_bar }
+					onChange={ ( newValue ) => {
+						updateFields( { wpcom_hide_action_bar: newValue } );
+					} }
+				/>
+				<FormSettingExplanation>
+					<a href="https://en.support.wordpress.com/action-bar/">
+						{ translate( 'Learn more about the Action Bar.' ) }
```

Comment:
Similarly, check how this privacy link opens the help center. It would be nice if this one can do the same. 

See a comparison of the behavior of those two links in this video:


https://github.com/user-attachments/assets/9c575d4a-e9b2-46c5-bdae-7a6850002427

Reply:
I made the suggested change but I'm not sure how to test it successfully. I tried different locales but it's very probable the specific page has no translations yet (or at least the languages I tried)? 🤔

Reply:
I just checked, and you're right: that page has never been translated into other languages.

cc @Automattic/i18n and @dlind1 - do we want to translate it, or is it intentionally left out?

Reply:
That wasn't intentional, but also not an oversight since not every page is automatically being translated :)
I've added it to the queue and it should be sent for translation on Monday the 20th.